### PR TITLE
Wait for a maximum of 60 minutes for logread to start. If it does not…

### PR DESCRIPTION
**Pull request by gkodali-zededa**
… start in 60 minutes reboot the device

Signed-off-by: GopiKrishna Kodali <gkodali@zededa.com>

### CODE REVIEW

File: pkg/pillar/scripts/device-steps.sh [hunk 1]
```diff
@@ -54,6 +55,18 @@ else
     USE_HW_WATCHDOG=0
 fi
 
+LOOP_COUNT=0
+while [ -z $(pgrep logread) ];
+do
+    sleep 1
+    LOOP_COUNT=`expr $LOOP_COUNT + 1`
+    if [ $LOOP_COUNT -ge $LOGREAD_PID_WAIT ]; then
+        echo "Error: Could not find logread process"
```
Comment (eriknordmark) [inline pkg/pillar/scripts/device-steps.sh]: Nit, but it would be good to have the timestamp here as we have for the other echo log commands.

Comment (gkodali-zededa) [reply pkg/pillar/scripts/device-steps.sh]: Addressed your comment and committed code.

**Comment by rvs** [comment]
Hey @gkodali-zededa -- since you have to take care of yetus feedback anyway, can you please fold this into #537 so that this PR can be closed/killed and instead #537 applies cleanly?

**Comment by deitch** [comment]
Can we please not? This is a small, single fix, on a single file, the optimal PR, and I am blocked on a few other things for this.

**Comment by deitch** [comment]
In any case, 4 tiny yetus changes.

**Comment by deitch** [comment]
https://7558-182198941-gh.circle-artifacts.com/0/tmp/yetus-out/diff-patch-shellcheck.txt

**Comment by gkodali-zededa** [comment]
Addressed the yetus issue already and am waiting for the build to finish.

**Comment by rvs** [comment]
Fair enough @deitch -- merging this now

**Pull request merged**